### PR TITLE
added audit fields

### PR DESCRIPTION
**Pull request by Leeyaoi**

### CODE REVIEW

File: InnoViber.DAL/Chat.cs [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-        public Guid Id { get; set; }
-        public string Name { get; set; } = string.Empty;
-        List<User> Users { get; set; } = new();
-        List<Message> Messages { get; set;} = new();
-    }
+    public Guid Id { get; set; }
+    public string Name { get; set; } = string.Empty;
+    List<User> Users { get; set; } = new();
+    List<Message> Messages { get; set;} = new();
+    public DateTime createdAt { get; set; }
```
Comment (moltenessence) [inline InnoViber.DAL/Chat.cs]: why createdAt / updatedAt namings in lower case?

File: InnoViber.DAL/IAuditable.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace InnoViber.DAL;
+internal interface IAuditable
+{
```
Comment (moltenessence) [inline InnoViber.DAL/IAuditable.cs]: formatting

File: InnoViber.DAL/IAuditable.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    public DateTime createdAt {  get; set; }
+    public DateTime updatedAt { get; set; }
```
Comment (moltenessence) [inline InnoViber.DAL/IAuditable.cs]: lower case

**Review by moltenessence — CHANGES_REQUESTED** [review]

**Review by moltenessence — APPROVED** [review]

File: InnoViber.DAL/User.cs [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
-        public List<Message> Messages { get; set; } = new();
-        public List<Chat> Chats { get; set; } = new();
-    }
+    public Guid Id { get; set; }
+    public string Name { get; set; } = string.Empty;
+    public string Surname { get; set; } = string.Empty;
+    public string Email { get; set; } = string.Empty;
+    public List<Message> Messages { get; set; } = new();
+    public List<Chat> Chats { get; set; } = new();
+    public DateTime CreatedAt { get; set; }
```
Comment (pavelGagarin0870) [inline InnoViber.DAL/User.cs]: Why it needed if exist in interface

File: InnoViber.DAL/User.cs [hunk 5]
```diff
@@ -1,14 +1,15 @@
...
-    public class User
-    {
-        public Guid Id { get; set; }
-        public string Name { get; set; } = string.Empty;
-        public string Surname { get; set; } = string.Empty;
-        public string Email { get; set; } = string.Empty;
-        public List<Message> Messages { get; set; } = new();
-        public List<Chat> Chats { get; set; } = new();
-    }
+    public Guid Id { get; set; }
```
Comment (pavelGagarin0870) [inline InnoViber.DAL/User.cs]: We can also move Id to interface

**Review by pavelGagarin0870 — CHANGES_REQUESTED** [review]

**Review by moltenessence — APPROVED** [review]

**Review by pavelGagarin0870 — APPROVED** [review]

**Pull request merged**